src/main/java/internet/shop/service/impl/ProductServiceImpl.java

```diff
@@ -0,0 +1,39 @@
+package internet.shop.service.impl;
+
+import internet.shop.dao.daointerface.ProductDao;
+import internet.shop.lib.Inject;
+import internet.shop.lib.Service;
+import internet.shop.model.Product;
+import internet.shop.service.serviceinterface.ProductService;
+import java.util.List;
+
+@Service
+public class ProductServiceImpl implements ProductService {
+    @Inject
+    private ProductDao productDao;
+
+    @Override
+    public Product create(Product product) {
+        return productDao.create(product);
+    }
+
+    @Override
+    public Product get(Long id) {
+        return productDao.get(id).orElseThrow();
```

Comment:
let's use `get()` for now
```suggestion
        return productDao.get(id).get();
```